scripts/Tools/testreporter.py

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env python
+
+"""
+
+Simple script to populate CESM test database with test results.  This can be 
+run with out any knowledge of CIME.
+
+"""
+
+import os
+import glob
+import xml.etree.ElementTree as ET
+from xml.dom import minidom
+import HTMLParser 
+import urllib
+import argparse
+
+
+
+# Parse command line options
+
+parser = argparse.ArgumentParser(description='Arguements for testreporter')
+parser.add_argument("--tagname", 
+                        help="Name of the tag being tested.")
+parser.add_argument("--testid", 
+                        help="Test id, ie c2_0_a6g,c2_0_b6g.")
+parser.add_argument("--testroot", 
+                        help="Root directory for tests to populate the database.")
+parser.add_argument("--testtype", 
+                        help="Type of test, prealpha or prebeta.")
+parser.add_argument("--dryrun",action="store_true", 
+                        help="Do a dry run, database will not be populated.")
+parser.add_argument("--dumpxml",action="store_true", 
+                        help="Dump XML test results to sceen.")
+args = parser.parse_args()
+
+# Fill in values needed from xml files (COMPILER, MPILIB, MACH, BASELINE_NAME_CMP.
+
+os.chdir(args.testroot)
+
+xml_file=glob.glob("*"+args.testid+"/env_build.xml")
```

Comment:
You should not use ET directly - if the CIME/XML interfaces are not adequate that's a bug in the interface that should be fixed.